controllers/topics.controllers.js

```diff
@@ -1,19 +1,38 @@
-const { pullTopics } = require("../models/topics.models");
+const {
+  pullTopics,
+  pullArticleById,
+  updateArticleId,
+} = require("../models/topics.models");
 
-exports.invalidEndPoint = (req,res,next) => {
-  let err = new Error("Invalid end point");
-  next(err);
-}
-
-exports.getTopics = (req,res,next) => {
-  console.log("controller start");
+exports.getTopics = (req, res, next) => {
   pullTopics()
-  .then((result)=>{
-    console.log("controller then");
-    res.status(200).send(result);
-  })
-  .catch((err)=>{
-    console.log("err controller", err);
-    next(err);
-  });
+    .then((result) => {
+      res.status(200).send(result);
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.getArticleById = (req, res, next) => {
```

Comment:
Got an article function inside the topics controller, would be nice to add all the article models into a new article model folder 😄

Reply:
Same for models!